src/episode.py

```diff
@@ -0,0 +1,36 @@
+from telegram.ext import Updater
+import requests
+from bs4 import BeautifulSoup
+from telegram.ext import CommandHandler
+
+# Define podcast formats
+urlHome = 'https://einundzwanzig.space'
+urlNews = 'https://einundzwanzig.space/podcast/news/'
+urlLese = 'https://einundzwanzig.space/podcast/lesestunde/'
+urlWeg = 'https://einundzwanzig.space/podcast/der-weg/'
+urlAll = 'https://einundzwanzig.space/podcast/'
+urlInterviews = 'https://einundzwanzig.space/podcast/interviews/'
+
+def getEpisode(url):
```

Comment:
I think this function needs some error handling. What happens if the request fails? Should we inform the user?